src/modules/checkbox.js

```diff
@@ -0,0 +1,17 @@
+export default function checkTodos(todo) {
+  const todos = JSON.parse(localStorage.getItem('myList'));
+  const todoIndex = todos.findIndex((item) => item.description === todo.children[1].value);
```

Comment:
- [ ]  Kindly ensure `arrow functions ` are used instead of the `function keyword` as I have given an explanation above